docs/source/common/test_set.rst

```diff
@@ -1,17 +1,26 @@
 .. _test_set:
 
-Test set
-========
-Lightning forces the user to run the test set separately to make sure it isn't evaluated by mistake.
-Testing is performed using the ``trainer`` object's ``.test()`` method.
+##########
+Evaluation
+##########
+
+*******
```

Comment:
I think we should start by best practices around validation vs test and then introduce how to use them to perform evaluation.